Formula/o/onion-location.rb

```diff
@@ -0,0 +1,44 @@
+class OnionLocation < Formula
+  include Language::Python::Virtualenv
+
+  desc "Discover advertised Onion-Location for given URLs"
+  homepage "https://codeberg.org/Freso/python-onion-location"
+  url "https://files.pythonhosted.org/packages/72/0d/e2656bdb8c66dc590da40622ca843f0513cd6f4b78bb1f9b6ed4592d283e/onion_location-0.1.0.tar.gz"
+  sha256 "37dc14eab3a22b8948f8301542344144682108d1564289482827dc45106ee1d5"
+  license "AGPL-3.0-or-later"
+  head "https://codeberg.org/Freso/python-onion-location.git", branch: "main"
+
+  depends_on "python@3.12"
+
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+
+  resource "beautifulsoup4" do
+    url "https://files.pythonhosted.org/packages/b3/ca/824b1195773ce6166d388573fc106ce56d4a805bd7427b624e063596ec58/beautifulsoup4-4.12.3.tar.gz"
+    sha256 "74e3d1928edc070d21748185c46e3fb33490f22f52a3addee9aee0f4f7781051"
+  end
+
+  resource "bs4" do
+    url "https://files.pythonhosted.org/packages/c9/aa/4acaf814ff901145da37332e05bb510452ebed97bc9602695059dd46ef39/bs4-0.0.2.tar.gz"
+    sha256 "a48685c58f50fe127722417bae83fe6badf500d54b55f7e39ffe43b798653925"
+  end
+
+  resource "lxml" do
+    url "https://files.pythonhosted.org/packages/63/f7/ffbb6d2eb67b80a45b8a0834baa5557a14a5ffce0979439e7cd7f0c4055b/lxml-5.2.2.tar.gz"
+    sha256 "bb2dc4898180bea79863d5487e5f9c7c34297414bad54bcd0f0852aee9cfdb87"
+  end
+
+  resource "soupsieve" do
+    url "https://files.pythonhosted.org/packages/ce/21/952a240de1c196c7e3fbcd4e559681f0419b1280c617db21157a0390717b/soupsieve-2.5.tar.gz"
+    sha256 "5663d5a7b3bfaeee0bc4372e7fc48f9cff4940b3eec54a6451cc5299f1097690"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    assert_match "http://2gzyxa5ihm7nsggfxnu52rck2vv4rvmdlkiu3zzui5du4xyclen53wid.onion/index.html",
+      shell_output(bin/"onion-location https://www.torproject.org/")
```

Comment:
`bin/"onion-location https://www.torproject.org/"` is a single `Pathname` which doesn't look correct here.

```suggestion
      shell_output("#{bin}/onion-location https://www.torproject.org/")
```

Reply:
yeah, make sense 👍